lesson14.html

```diff
@@ -0,0 +1,161 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tooltips</title>
+    <style>
+      .container_blue {
+        display: flex;
+        flex-direction: row;
+        gap: 50px;
+        padding: 20px;
+        background-color: white;
+        border: 5px solid blue;
+        width: 90%;
+        opacity: 0;
+        animation: slideIn 1s forwards 1s;
+      }
+      .trigger {
+        background-color: white;
+        padding: 10px;
+        border: 2px solid red;
+        position: relative;
+        display: inline-block;
+      }
+      @keyframes slideIn {
+        0% {
+          transform: translateX(-100%);
+          opacity: 0;
+        }
+        100% {
+          transform: translateX(0);
+          opacity: 1;
+        }
+      }
+      .tooltiptext {
+        visibility: hidden;
+        opacity: 0;
+        transition: opacity 1s ease, visibility 1s ease;
+        background-color: gray;
+        color: black;
+        text-align: center;
+        border-radius: 9px;
+        position: absolute;
+        z-index: 1;
+      }
+      .tooltip1 .tooltiptext {
+        top: 50%;
+        width: 120px;
+        padding: 5px 10px;
+        left: 0;
+        transform: translate(-15%, -140%);
+      }
+      .tooltip1:hover .tooltiptext,
+      .tooltip2:hover .tooltiptext,
+      .tooltip3:hover .tooltiptext,
+      .tooltip4:hover .tooltiptext {
+        visibility: visible;
+        opacity: 1;
+      }
+      .tooltip1 .tooltiptext::after,
+      .tooltip2 .tooltiptext::after,
+      .tooltip3 .tooltiptext::after,
+      .tooltip4 .tooltiptext::after {
+        content: " ";
+        position: absolute;
+      }
+      .tooltip1 .tooltiptext::after,
+      .tooltip3 .tooltiptext::after {
+        border-left: 10px solid transparent;
+        border-right: 10px solid transparent;
+        transform: translateX(-50%);
+      }
+      .tooltip1 .tooltiptext::after {
+        top: 100%;
+        left: 50%;
+        border-top: 10px solid gray;
+      }
+      .tooltip2 .tooltiptext {
+        top: -50%;
+        width: 120px;
+        padding: 9px;
+        left: 0;
+        transform: translateX(77%);
+      }
+      .tooltip2 .tooltiptext::after,
+      .tooltip4 .tooltiptext::after {
+        top: 50%;
+        border-bottom: 10px solid transparent;
+        border-top: 10px solid transparent;
+        transform: translateY(-50%);
+      }
+      .tooltip2 .tooltiptext::after {
+        left: -10px;
+        border-right: 10px solid gray;
+      }
+      .tooltip3 .tooltiptext {
+        top: 110%;
+        width: 110px;
+        padding: 10px 20px;
+        left: 50%;
+        transform: translateX(-50%);
+      }
+      .tooltip3 .tooltiptext::after {
+        top: -10%;
+        left: 80px;
+        border-bottom: 10px solid gray;
+      }
+      .tooltip4 .tooltiptext {
+        transform: translateY(-50%);
+        top: 50%;
+        right: 109%;
+        width: 120px;
```

Comment:
I believe you can make all ```tooltiptext``` look the same (width, padding)
and change only their position (transform, top, right)